app/templates/signin.hbs

```diff
@@ -1,33 +1,43 @@
 <TravisLayout @layoutName="layouts/center">
   <div class="content-page">
-    <img
-      src="/images/logos/TravisCI-Mascot-1.svg"
-      alt="Travis CI mascot"
-      width="200"
-      height="200"
-    />
+    {{#if this.isRedirectingToAccountPage}}
+      <img
+        src="/images/billing/account-unauthorized.svg"
+        alt="Account"
+        width="350"
+        height="350"
+      />
```

Comment:
You can probably replace this with an `SvgImage` component if you wanted. It would be like
```
<SvgImage
  @name='account-unauthorized'
  @width={{350}}
  @height={{350}}
  @alt='Account'
/>
```
It's not required though, just an option for working with SVGs in `travis-web` that I find kind of nice. You can see all SVGs available along with what name you can use to reference them at http://localhost:4200/ember-svg-jar/index.html while developing. Although it seems like `TravisCI-Mascot-1.svg` doesn't show up in there for some reason, so probably best to leave that one as-is.

Reply:
Awesome! :tada: Yeah I was very happy when I found out about it, it's very handy.